workbench-core/accounts/src/services/projectService.ts

```diff
@@ -170,11 +178,16 @@ export default class ProjectService {
    * @returns Project object of new project
    */
   public async createProject(params: CreateProjectRequest, user: AuthenticatedUser): Promise<Project> {
-    // Verify caller is an IT Admin--TODO implement after dynamic AuthZ--this can happen in the middleware during integration but keeping here so this code gets moved properly
-    // const userGroupsForCurrentUser: string[] = await this._dynamicAuthorizationService.getUserGroups(user.id);
-    // if(userGroupsForCurrentUser.length !== 1 || userGroupsForCurrentUser[0] !== 'ITAdmin') {
-    //   throw Boom.forbidden('Only IT Admins are allowed to create new Projects.');
-    // }
+    const getUserGroupsResponse = await this._dynamicAuthorizationService.getUserGroups({
+      authenticatedUser: user,
+      userId: user.id
+    });
+
+    const userGroupsForCurrentUser = getUserGroupsResponse.data.groupIds || [];
+
+    if (!userGroupsForCurrentUser.includes('ITAdmin')) {
```

Comment:
This is different than the original implementations. Make sure to talk about implications

Reply:
Good point! When we run integ tests, do we have admin user? I remember that we do create projects in tests

Reply:
yes as long as you've run the post-deploy script

Reply:
This check can happen in middleware so we may not need it here

Reply:
i think we want to keep it because we need to do different logic based on the role.

Reply:
Not for create. Check whats under this if statement.

Reply:
+1 regulation of ITAdmins being the only ones who can create projects is managed by route middleware. This lets partners enable project creation for other roles if necessary (if some customer wanted researchers to be able to make their own projects).

Reply:
This is the LIST endpoint, not create

Reply:
No its not. i'm an idiot.